preview/src/index.html

```diff
@@ -29,7 +29,7 @@
 <body>
     <div id="nav-bar"></div>
     <div id="flow-editor"></div>
-    <script src="floweditor.js"></script>
+    <script src="flow-editor.js"></script>
```

Comment:
idk if you were running into this problem, but i wasn't able to run the app in development because the filename we were outputting was `@nyaruka/flow-editor.js`...seems like an oversight on my part...should have been addressed by https://github.com/nyaruka/floweditor/pull/283